aspnetcore/grpc/configuration.md

```diff
@@ -0,0 +1,82 @@
+---
+title: gRPC for ASP.NET Core configuration
+author: jamesnk
+description: Learn how to configure gRPC for ASP.NET Core apps.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: jamesnk
+ms.custom: mvc
+ms.date: 04/09/2019
+uid: grpc/configuration
+---
+# gRPC for ASP.NET Core configuration
+
+## Configure services options
+
+The following table describes options for configuring gRPC services:
+
+| Option | Default Value | Description |
+| ------ | ------------- | ----------- |
+| `SendMaxMessageSize` | `null` | The maximum message size in bytes that can be sent from the server. Attempting to send a message that exceeds the configured maximum message size will result in an exception. |
```

Comment:
| `SendMaxMessageSize` | `null` | The maximum message size in bytes that can be sent from the server. Attempting to send a message that exceeds the configured maximum message size results in an exception. |